### ISSUE TITLE
Add support for `Element.firstElementChild` property

### ISSUE THREAD

**Issue by piranna** [xmldom/xmldom#328]
https://developer.mozilla.org/en-US/docs/Web/API/Element/firstElementChild

The `Element.firstElementChild` read-only property returns an element's first child `Element`, or `null` if there are no child elements.

`Element.firstElementChild` includes only element nodes. `xmldom` only implements `Node.firstChild`, that return all the child nodes, including non-element nodes like text and comment nodes.

**Comment by karfau** [comment]
I guess talking about the category of methods in this realm would mean that  people would also expect lastElementChild (if something like that exists, didn't check) and potentially other accessors.

It will not be the highest prio by the maintainers, but it will also not be rejected.

It's related to the discussion regarding supporting more thing from the living standard and also regarding providing the correct actual types instead of relying on the ones from typescript. (Will link the tickets later.)

**Comment by piranna** [comment]
Yes, ideally it should implement the other `Element` class APIs, too.

**Comment by KevinYeramian** [comment]
Hey @karfau,
I would be very interested to work on this. The issue is quite old, is it something that is still "not rejected" by the maintainers?

**Comment by karfau** [comment]
@KevinYeramian anything that is aligned with the spec and is not requiring major rewrites or new dependencies is always welcome.

You can always expect feedback and suggestions how to improve the contribution, but most likely no substantial code contributions by maintainers.
And we will expect unit tests for every new API including those for edge cases.
Ideally you can provide small chunks of changes/multiple focused PRs, so reviewing each is easier for us.
In this case maybe adding useful pairs of APIs.

**Comment by KevinYeramian** [comment]
@karfau thank you for the answer.

I just create the PR #398.
